packages/neuron-ui/src/components/UANDisplay/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styles from './index.module.scss'
+
+const symbolReg = /^([A-Z]|[0-9])+\.([a-z]|[0-9])+(\|([a-z]|[0-9])+\.([a-z]|[0-9])+)*$/
```

Comment:
We cannot assume that symbol follows `xxx.xxx.xxx` because name can be chained infinitely.

But we can simply truncate the first fragment because `.` is a special char used in `UAN`

Reply:
`<asset> ::= <asset-symbol> "." <chain-symbol>`
It seems `asset` can only include one `.`.

Reply:
Got it